builds/Makefile

```diff
@@ -2075,10 +2075,10 @@ TV_CROSS_CONFIGURE_FLAGS=$(CROSS_BASE_CONFIGURE_FLAGS)	\
 	--prefix=$(BUILD_DESTDIR)/crosstv \
 	--cache-file=../crosstv.config.cache \
 	--target=aarch64-unknown-darwin \
+	--with-llvm=../llvm64/usr
 
 TV_CROSS_CONFIGURE_ENVIRONMENT = \
-	$(Q) PATH="../llvm64/usr/bin:$(PATH)" \
-	CFLAGS="$(TV_CROSS_CFLAGS)" \
```

Comment:
Ah, the diff confused me, I though the first line was changed, not the second. NVM then, everything is good :smile: